charmtools/build/builder.py

```diff
@@ -675,16 +675,19 @@ def main(args=None):
 
         build()
 
-        lint, exit_code = proof.proof(os.getcwd(), False, False)
-        if lint:
-            llog = logging.getLogger("proof")
-            for line in lint:
-                if line[0] == "I":
-                    llog.info(line)
-                elif line[0] == "W":
-                    llog.warn(line)
-                elif line[0] == "E":
-                    llog.error(line)
+        lint, exit_code = proof.proof(build.target_dir, False, False)
+        llog = logging.getLogger("proof")
+
+        if not lint:
+            llog.info('OK!')
+
+        for line in lint:
```

Comment:
what if (one day) `lint` is none?  you would be laughed at.  please add an else.